src/swagger-specs/destinations.yaml

```diff
@@ -34,81 +34,313 @@ servers:
     description: Adobe IO gateway endpoint
 
 tags:
-
+  - name: Changelog
+    description: This section includes the changes introduced to the Flow Service - Destinations API.
+            <table><thead><tr><th>Release date</th><th>Parameter or method update</th><th>Breaking change (Yes/No)</th><th>Description</th></tr></thead>
+            <tbody>
+            <tr><td>09/07/2022</td><td>`isDestinationFlow` parameter</td><td>No</td><td>This parameter is returned in API responses for destination dataflows only. <br> Use it when calling the <b>Retrieve a list of destination dataflows GET endpoint</b> with the query parameter `property=inheritedAttributes.properties.isDestinationFlow==true` to return destination dataflows only.</td></tr>
+            <tr><td>09/07/2022</td><td>Published Destinations Flow Service - API reference</td><td>No</td><td>Initial publish of the API reference documentation to the Adobe Experience Platform developer website.</td></tr>
+            </tbody></table>
```

Comment:
Should we be tracking updates to APIs like this? I'd be worried about duplicating between here and the Platform release notes, which we should be linking to.

Reply:
I agree with your point and I will comment out the changelog for now. 

That said, taken from customer feedback, we will need to think about how to improve the way we communicate API updates in our release notes. I will certainly want to mention this new API reference in this month's release notes.